Description: format error in man page
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Forwarded: not-needed
Last-Update: 2016-08-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/doc/sane-mustek_usb.man
===================================================================
--- trunk.orig/doc/sane-mustek_usb.man
+++ trunk/doc/sane-mustek_usb.man
@@ -180,6 +180,7 @@ export SANE_DEBUG_MUSTEK_USB=4
 .BR sane\-ma1509 (5)
 .br
 .IR @DOCDIR@/mustek_usb/mustek_usb.CHANGES ,
+.br
 .I @DOCDIR@/mustek_usb/mustek_usb.TODO
 .br
 .I http://www.meier\-geinitz.de/sane/mustek_usb\-backend/
Index: trunk/doc/sane-genesys.man
===================================================================
--- trunk.orig/doc/sane-genesys.man
+++ trunk/doc/sane-genesys.man
@@ -52,7 +52,7 @@ in the feeder then start calibration eit
 or by clicking on the available 'calibrate' button in the 'advanced options' in a graphical
 frontend. The result of the calibration is stored in a file in the home directory of the user doing it.
 If you plug the scanner in another machine or use it with another account, calibration
-will have to be redone, unles you use the \-\-calibration\-file option.
+will have to be redone, unless you use the \-\-calibration\-file option.
 If no home directory is defined,
 USERAPPPROFILE will be used, then TMPDIR or TMP. If none of these directories exist, the backend will try
 to write in the current working directory. Flatbed scanners also make use of the calibration file as a cache
@@ -71,7 +71,7 @@ A value of -1 means forever, 0 means no
 .B \-\-threshold percent
 .RS
 	0..100% (in steps of 1). Select minimum brightness to get a white point. Pixels
-whith brightness below that value will be scanned as black.
+with brightness below that value will be scanned as black.
 .RE
 
 .B \-\-brightness value
Index: trunk/doc/saned.man
===================================================================
--- trunk.orig/doc/saned.man
+++ trunk/doc/saned.man
@@ -226,7 +226,7 @@ compiled in, but then logging of debug i
 The systemd configuration is different for the 2 options, so
 both are described below.
 .SH Systemd configuration for saned with systemd support compiled in
-for the systemd configuration we need to add 2 configuation files in 
+for the systemd configuration we need to add 2 configuration files in 
 .I /etc/systemd/system.
 .PP
 The first file we need to add here is called 
@@ -301,7 +301,7 @@ forwarded to the system log.
 This configuration will also work when Saned is compiled WITH systemd integration
 support, but it does not allow debugging information to be logged.
 .PP
-for systemd configuration for saned, we need to add 2 configuation files in
+for systemd configuration for saned, we need to add 2 configuration files in
 .I /etc/systemd/system.
 .PP
 The first file we need to add here is called
Index: trunk/doc/sane-epson.man
===================================================================
--- trunk.orig/doc/sane-epson.man
+++ trunk/doc/sane-epson.man
@@ -77,7 +77,7 @@ The
 option selects the bit depth the scanner is using. This option is only
 available for scanners that support more than one bit depth. Older
 scanners will always transfer the image in 8bit mode. Newer scanners
-allow to select either 8 bits, 12 or 14 bits per color channel. For a
+allow one to select either 8 bits, 12 or 14 bits per color channel. For a
 color scan this means an effective color depth of 36 or 42 bits over
 all three channels. The valid choices depend on the scanner model.
 
Index: trunk/doc/sane-epson2.man
===================================================================
--- trunk.orig/doc/sane-epson2.man
+++ trunk/doc/sane-epson2.man
@@ -123,7 +123,7 @@ The
 option selects the bit depth the scanner is using. This option is only
 available for scanners that support more than one bit depth. Older
 scanners will always transfer the image in 8bit mode. Newer scanners
-allow to select either 8 bits, 12 or 14 bits per color channel. For a
+allow one to select either 8 bits, 12 or 14 bits per color channel. For a
 color scan this means an effective color depth of 36 or 42 bits over
 all three channels. The valid choices depend on the scanner model.
 
Index: trunk/doc/sane-epsonds.man
===================================================================
--- trunk.orig/doc/sane-epsonds.man
+++ trunk/doc/sane-epsonds.man
@@ -29,7 +29,7 @@ The
 option selects the bit depth the scanner is using. This option is only
 available for scanners that support more than one bit depth. Older
 scanners will always transfer the image in 8bit mode. Newer scanners
-allow to select either 8 bits, 12 or 14 bits per color channel. For a
+allow one to select either 8 bits, 12 or 14 bits per color channel. For a
 color scan this means an effective color depth of 36 or 42 bits over
 all three channels. The valid choices depend on the scanner model.
 
